airflow/providers/apache/livy/operators/livy.py

```diff
@@ -207,4 +207,5 @@ def execute_complete(self, context: Context, event: dict[str, Any]) -> Any:
             self.task_id,
             event["response"],
         )
+        context["ti"].xcom_push(key="app_id", value=self.get_hook().get_batch(event["batch_id"])["appId"])
```

Comment:
i notice that you aren't defensive here e.g. 
```
event["batch_id"])["appId"]
```

instead of perhaps

```
event.get("batch_id", {}).get("appId", None)
```
presumably you've verified that this is never an issue?